### ISSUE TITLE
Can i mine with wallet same as pool address ?

### ISSUE THREAD

**Issue by p2svn** [Snipa22/nodejs-pool#252]
Can i mine with wallet same as pool address ?

**Comment by vitorgatti** [comment]
with the fee address yes I think so

with the pool wallet I don't think so, the payment would get an error (trying to pay to itself)

**Comment by bobbieltd** [comment]
Yeah, I like your answer. You can test by sending XMR from your wallet to your same wallet. If it pass then answer Yes, if not then answe No.

**Comment by bobbieltd** [comment]
Both pool and fee wallet, the answer is NO.

**Comment by p2svn** [comment]
the problem here is although i mine with different address
"Invalid miner, disconnecting due to: Invalid payment      │"
How to now what miner with invalid address ?

**Comment by Snipa22** [comment]
Looks like an invalid payment address, which means it didn't conform to the coin's spec: https://github.com/Snipa22/nodejs-pool/blob/master/lib/pool.js#L299

Technically, the pool will let you mine with whatever address you want, pool/fee/etc.  I don't bother to verify the address against the pool's addresses, but it's a tad silly to do so, because of the transaction loop.  The above code is what handles the verification of addresses during login.
